exporter/googlemanagedprometheusexporter/config.go

```diff
@@ -38,14 +38,19 @@ type Config struct {
 type GMPConfig struct {
 	ProjectID    string                 `mapstructure:"project"`
 	UserAgent    string                 `mapstructure:"user_agent"`
+        // MetricPrefix configures the prefix of metrics sent to GoogleManagedPrometheus.  Defaults to prometheus.googleapis.com.
+        // Changing this prefix is not recommended, as it may cause metrics to not be queryable with promql in the Cloud Monitoring UI.
+	MetricPrefix string                 `mapstructure:"metric_prefix"`
```

Comment:
should we just make this `prefix` to match the googlecloud exporter? https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/exporter/collector/config.go#L86.

We could even consider nesting it under metric to make the config surfaces similar.

WDYT?

Reply:
SG. Just pushed a new commit to make a nested `metric.prefix`, also moved ClientConfig under metric to be consistent with googlecloud exporter.